src/resources/postcss/views/skeleton/_single-event.pcss

```diff
@@ -4,9 +4,28 @@
  * @package TribeEventsCalendar
  *
  */
-	
+ 
+/* Hide a bunch of elements (from v1 styles) */
```

Comment:
are we still hiding v1 styles in v2? or are we removing v1 styles?

Reply:
@paulmskim those are some elements that were hidden in v1 and they're still not appearing in the v2 designs, even though they're part of the HTML output

Reply:
Gotcha, sounds good.